packages/typespec-python/src/external-process.ts

```diff
@@ -21,6 +21,7 @@ export async function saveCodeModelAsYaml(name: string, codemodel: unknown): Pro
     await mkdir(cadlCodeGenTempDir, { recursive: true });
     const filename = createTempPath(".yaml", name);
     const yamlStr = jsyaml.dump(codemodel);
+    await writeFile(joinPaths("C:/dev/autorest.python/packages/typespec-python", "alpha", "output.yaml"), yamlStr);
```

Comment:
@msyyc is this your debug code?

Reply:
Sorry to commit it and I will revert this code.